packages/blade/src/components/Button/Button.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import Button from './Button';
+
+const ButtonMeta: ComponentMeta<typeof Button> = {
+  title: 'Components/Button',
```

Comment:
Why such a title? Does it create a hierarchy in the menu?

Reply:
yes